cog.c

```diff
@@ -284,6 +289,8 @@ on_handle_local_options (GApplication *application,
                                                          s_options.web_extensions_dir);
     }
 
+    g_object_set (shell, "ignore-tls-errors", s_options.ignore_tls_errors, NULL);
```

Comment:
I would rather add a `cog_shell_set_ignore_tls_errors()` setter and use it here instead.